resources/js/Pages/Adminland/Employee/Archives/Show.vue

```diff
@@ -242,24 +257,55 @@ export default {
 
   data() {
     return {
-      loadingState: '',
+      dataReport: null,
+      form: useForm(),
+      refresh: _.debounce(() => this.doRefresh(), 1000),
     };
   },
 
+  computed: {
+    localReport() {
+      return this.dataReport ?? this.report;
+    }
+  },
+
+  mounted() {
+    this.dataReport = this.report;
+    this.refresh();
+  },
+
+  unmounted() {
+    this.refresh.cancel();
+  },
+
   methods: {
     submit() {
-      this.loadingState = 'loading';
-
-      axios.post(`/${this.$page.props.auth.company.id}/account/employees/upload/archives/${this.report.id}/import`)
-        .then(response => {
+      axios.post(this.route('account.employees.upload.archive.import', {
+        company: this.$page.props.auth.company.id,
+        archive: this.report.id
+      }))
+        .then(() => {
           localStorage.success = this.$t('account.import_employees_import_success');
-          this.$inertia.visit(`/${this.$page.props.auth.company.id}/account/employees/upload/archives`);
-        })
-        .catch(error => {
-          this.loadingState = null;
-          this.form.errors = error.response.data;
+          this.$inertia.visit(this.route('account.employees.upload.archive', {company: this.$page.props.auth.company.id}));
         });
     },
+
+    doRefresh() {
```

Comment:
Interesting. So you are polling and don't rely on a web socket for this. I'm not against this by the way, it works. But here there is still a problem:

when you load the Show page, regardless of the job being already done, you will refresh the page every second. This should'nt load every second if the job has a different state than `importing` (or else).

Reply:
It's done some line below: the refresh is not done if the status is 'imported' or 'uploaded'